.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: go
-go: 1.8.3
+go: '1.10'
 sudo: false
 install: true
 script:
-- go get -u -v ./...
-- go test -v ./...
-- go test -v -race ./...
\ No newline at end of file
+- go test -v -race ./...
```

Comment:
Does travis cache anything? If so we may need to add `-count=1` to ensure that we don't have a test that skirts cache busting.